packages/@aws-cdk/toolkit/lib/actions/watch/index.ts

```diff
@@ -38,11 +38,11 @@ export interface WatchOptions extends BaseDeployOptions {
   readonly watchDir?: string;
 
   /**
-   * The output to write CloudFormation template to
+   * The output directory to write CloudFormation template to
    *
    * @deprecated this should be grabbed from the cloud assembly itself
    */
-  readonly output?: string;
+  readonly outdir?: string;
```

Comment:
We previously used a mix in names. This is aligning on `outdir` as the canonical short version of "**Out**put **Dir**ectory"